mongo/options/bulkwriteoptions.go

```diff
@@ -6,37 +6,41 @@
 
 package options
 
-// DefaultOrdered is the default order for a BulkWriteOptions struct created from BulkWrite.
+// DefaultOrdered is the default value for the Ordered option in BulkWriteOptions.
```

Comment:
This was done because you can't do `&true`, so a constant was declared. It probably shouldn't have been exported, but we can't change that now. The description of the actual default value is in the `BulkWriteOptions` struct docs and I didn't want to duplicate it here.